server/executor/cli.cpp

```diff
@@ -11,16 +11,26 @@
 
 #include <rdmalib/rdmalib.hpp>
 #include <rdmalib/server.hpp>
+#include <rdmalib/libraries.hpp>
 #include "rdmalib/connection.hpp"
 #include "server.hpp"
 #include "fast_executor.hpp"
+#include "opts.hpp"
 
 int main(int argc, char ** argv)
 {
   // Register a SIGINT handler so that we can gracefully exit
   //server::SignalHandler sighandler;
 
-  auto opts = server::opts(argc, argv);
+
+  #ifdef USE_LIBFABRIC
```

Comment:
It would be better for this to be a runtime switch - we can compile with both implementations enabled.